src/mapDataManipulation/manipulateGain.ts

```diff
@@ -0,0 +1,46 @@
+import { mapDataManipulation } from 'src/mapDataManipulation/mapDataManipulation';
+
+// The algorithm works with numbers between 0 and 1, so we must:
+// - change the interval of the values from [0, 255] to [0, 1]
```

Comment:
Can we add a check that throws an error if the interval of the values is not between 0 and 255?

Reply:
Talked in-person, decided to simply limit to 0 and 255.

Reply:
resolving so i can merge